src/Comparator/BackwardsCompatibility/PropertyBased/PropertyDefaultValueChanged.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\ApiCompare\Comparator\BackwardsCompatibility\PropertyBased;
+
+use Roave\ApiCompare\Change;
+use Roave\ApiCompare\Changes;
+use Roave\BetterReflection\Reflection\ReflectionProperty;
+
+final class PropertyDefaultValueChanged implements PropertyBased
+{
+    public function compare(ReflectionProperty $fromProperty, ReflectionProperty $toProperty) : Changes
+    {
+        if ($fromProperty->isPrivate()) {
```

Comment:
Can this be extracted like `AccessibleMethodFunctionBasedChange` ?

Reply:
Most likely, yes. And good call too :+1: